samples/nextjs/src/components/Styleguide-FieldUsage-Number/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
@ambrauer Probably, you can this line from all component files. Next.js allow us to do it

Reply:
Unfortunately, this isn't actually on React v17 yet (b/c of `sitecore-jss-nextjs` package dependency on `sitecore-jss-react`), which this would require, correct? (https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). 
We should definitely discuss upgrading to React v17 for the whole repo separately.

Reply:
Nvm, you're right we can totally get rid of those. Done.